src/app/components/login/login.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container w-25">
+  <div class="row justify-content-center mt-5">
+    <h2 class="text-center mb-4">Bienvenido:</h2>
+    <form>
+      <div class="form-group">
+        <label for="usuario">Usuario:</label>
```

Comment:
```suggestion
        <label>Usuario:</label>
```
why u use the for attribute?